test_scripts/Defects/8_1/980_SDL_responds_GENERIC_ERROR_instead_of_INVALID_DATA_when_soft_button_has_Type_is_Image_or_Both_and_Text_is_whitespace_or_\t_or_\n_or_empty.lua

```diff
@@ -0,0 +1,81 @@
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/smartdevicelink/sdl_core/issues/980
+---------------------------------------------------------------------------------------------------
+-- Description: Check that SDL responds INVALID_DATA in case soft button has Type(Image or Both)
+-- and Text is whitespace or \t or \n or empty
+--
+-- Precondition:
+-- 1) AlertManeuver is allowed by policy
+-- 2) SDL, HMI, Mobile session started
+-- 3) App is registered and activated
+-- In case:
+-- 1) App sends AlertManeuver with soft button (type = Image or Both, Text = whitespace or \t or \n or empty)
+-- SDL does:
+-- 1) Respond with AlertManeuver(success = false, resultCode = "INVALID_DATA") to app
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require("user_modules/sequences/actions")
+local utils = require('user_modules/utils')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local requestParams = {
+  type_BOTH = {
+    ttsChunks = { { text = "FirstAlert", type = "TEXT" } },
+    softButtons = { { type = "BOTH", text = "Close", softButtonID = 821, } } -- soft button has type is "BOTH"
+  },
+  type_IMAGE = {
+    ttsChunks = { { text = "FirstAlert", type = "TEXT" } },
+    softButtons = { { type = "IMAGE", text = "Close", softButtonID = 821, } } -- soft button has type is "Image"
+  },
+  text_HorizontalTab = {
+    ttsChunks = { { text = "FirstAlert", type = "TEXT" } },
+    softButtons = { { type = "TEXT", text = "Close\t", softButtonID = 821, } } -- soft button has Text is \t
+  },
+  text_lineBreak = {
+    ttsChunks = { { text = "FirstAlert", type = "TEXT" } },
+    softButtons = { { type = "TEXT", text = "Close\n", softButtonID = 821 } } -- soft button has Text is \n
+  },
+   text_Whitespace = {
+    ttsChunks = { { text = "FirstAlert", type = "TEXT" } },
+    softButtons = { { type = "TEXT", text = "        ", softButtonID = 821 } } -- soft button has Text is whitespace
+  },
+  text_empty = {
+    ttsChunks = { { text = "FirstAlert", type = "TEXT" } },
+    softButtons = { { type = "TEXT", text = "", softButtonID = 821 } } -- soft button has Text is empty
```

Comment:
Seems the issue describes the following cases:
1. field `type` = `{IMAGE, BOTH}`
AND
2. field `text` = `{whitespace, \t, \n, empty}`

So all 8 combinations has to be covered